.github/workflows/push.yml

```diff
@@ -50,5 +50,5 @@ jobs:
           node-version: 14
       - name: Release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
I have to use my personal token otherwise the `on: release` workflow won't be triggered